src/components/File.vue

```diff
@@ -164,11 +179,13 @@ export default {
 		])
 
 		this.initialized = true
+
+		await this.$nextTick() // Wait for next tick to have the canvas in the DOM
```

Comment:
Could be prevented by adding a `BlurHash` component and move it to thats `mounted()` hook.
Might also a bit easier for maintenance when splitting this huge component?

Reply:
Definitely. I'll address code splitting in a following PR, let's keep this "simple" for now. :)